pkg/status/status_server.go

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2024 The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package status
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	// nolint
+	"google.golang.org/protobuf/encoding/protojson"
+
+	adminv2 "kmesh.net/kmesh/api/v2/admin"
+	"kmesh.net/kmesh/daemon/options"
+	"kmesh.net/kmesh/pkg/controller"
+	"kmesh.net/kmesh/pkg/controller/ads"
+	"kmesh.net/kmesh/pkg/logger"
+)
+
+var log = logger.NewLoggerField("status")
+
+const (
+	adminAddr = "localhost:15200"
+
+	patternHelp       = "/help"
+	patternOptions    = "/options"
+	patternBpfAdsMaps = "/debug/bpf/ads"
```

Comment:
@nlgwcy Should we separate ads and workload here?

Reply:
yes, need to be separated.